src/uart/uart.rs

```diff
@@ -0,0 +1,24 @@
+use crate::bits::BitManipulation;
+use crate::const_assert;
+use crate::memory_map::mmio;
+use crate::{bit_impl, reg_impl};
+use core::ptr;
+
+/// # UART Register Offsets
+/// See Max 78000 User Guide Pg 180, Table 12-7
```

Comment:
Include page range (ie. page 180 - XXX), and full `page` word to keep with project consistency.